blazor/dialog/predefined-dialogs.md

```diff
@@ -0,0 +1,338 @@
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-position.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-position.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-position.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert position Dialog](./images/blazor-alert-position.png)
+
+![Confirm position Dialog](./images/blazor-confirm-position.png)
+
+![Prompt position Dialog](./images/blazor-prompt-position.png)
+
+## Dimension
+
+You can customize the dialog dimensions using [DialogDimension.Height](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogDimension.html#Syncfusion_Blazor_Popups_DialogDimension_Height) and [DialogDimension.Width] properties. Use the following code to customize the dialog dimensions.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-dimension.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-dimension.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-dimension.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert dimension Dialog](./images/blazor-alert-dimension.png)
+
+![confirm dimension Dialog](./images/blazor-confirm-dimension.png)
+
+![prompt dimension Dialog](./images/blazor-prompt-dimension.png)
+
+## Close Button Dialog
+
+You can customize the close icon using [ShowCloseIcon](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialog.html#Syncfusion_Blazor_Popups_SfDialog_ShowCloseIcon) property. If the `ShowCloseIcon` property is set to `true` then the close icon will appear. Use the following code to enable the `ShowCloseIcon`.
+
+{% tabs %}
+
+{% highlight tabtitle="alert.razor" %}
+{% include_relative code-snippet/alert-close-button.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="confirm.razor" %}
+{% include_relative code-snippet/confirm-close-button.razor %}
+{% endhighlight %}
+
+{% highlight tabtitle="prompt.razor" %}
+{% include_relative code-snippet/prompt-close-button.razor %}
+{% endhighlight %}
+
+{% endtabs %}
+
+![Alert close icon Dialog](./images/blazor-alert-close-button.png)
+
+![Confirm close icon Dialog](./images/blazor-confirm-close-button.png)
+
+![Prompt close icon Dialog](./images/blazor-prompt-close-button.png)
+
+## Customization of Action Buttons
+
+You can customize the predefined dialog buttons by using [PrimaryButtonOptions](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogButton.html#Syncfusion_Blazor_Popups_DialogButton_IsPrimary) property. Use the following code to customize the predefined dialog buttons.
```

Comment:
You can customize the predefined dialog buttons by using the [PrimaryButtonOptions](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.DialogButton.html#Syncfusion_Blazor_Popups_DialogButton_IsPrimary) property. Use the following code to customize the predefined dialog buttons: